src/selectors/entities/apps.ts

```diff
@@ -4,14 +4,13 @@ import {GlobalState} from 'types/store';
 import {AppBinding} from 'types/apps';
 
 export function getAppsBindings(state: GlobalState, location?: string): AppBinding[] {
-    if (location) {
-        const bindings = state.entities.apps.bindings.find((p) => {
-            return p.location === location;
-        })?.bindings;
-        if (bindings) {
-            return bindings;
-        }
+    if (!state.entities.apps.bindings) {
         return [];
     }
+
+    if (location) {
+        const headerBindings = state.entities.apps.bindings.filter((b) => b.location === location);
```

Comment:
Why filter and not find? Does the server send more than one occurrence for the base location?

Reply:
@larkox Please see the description :) Yes the backend is returning one occurrence for each app, but I think that is likely not on purpose. cc @levb

Reply:
Yes, I did read the description, but what still puzzled by this :sweat_smile: Almost sure it is not on purpose.

Reply:
I defer to @mickmister to decide what is the hierarchy returned by the proxy; please avoid sending redundant data, especially if it's easy to do.